### ISSUE TITLE
[Templates] Override version

### ISSUE THREAD

**Issue by sandorfr** [geeklearningio/gl-swagger-generator#38]
For the `project` generation flavors which are likely to be published and consumed as package we should provide a way to set the version.

**Comment by asiffermann** [comment]
Agreed, I assumed the version would be patched before publication, but that's likely too much our way 😉
